docs/software/file_servers.md

```diff
@@ -2,13 +2,171 @@
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
 
-- [ProFTPD - Simple, efficient, lightweight FTP file server](https://dietpi.com/phpbb/viewtopic.php?p=55#p55)
-- [Samba - Feature rich file server](https://dietpi.com/phpbb/viewtopic.php?p=56#p56)
-- [vsFTPD - Feature rich FTP file server](https://dietpi.com/phpbb/viewtopic.php?p=2820#p2820)
-- [NFS - Network file system server](https://dietpi.com/phpbb/viewtopic.php?p=2821#p2821)
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## ProFTPD - Simple, efficient, lightweight FTP file server
+
+ProFTPD gives you access to files/music/downloads etc. on your DietPi system quickly and efficiently with minimal overhead.
+
+![DietPi file server software ProFTPD](../assets/images/dietpi-software-fileservers-proftpd.png)
+
+### Access to ProFTPD
+
+The access to ProFTPD with **Windows** is achieved as follows:
+
+- Go to My Computer (Windows Explorer).
+- In the address bar at the top type `ftp://dietpi:dietpi@192.168.0.100` press enter.
+
+Change 192.168.0.100 to the IP address of your DietPi system.
+
+The access to ProFTPD with an **FTP client** is achieved as follows:
+
+- username = dietpi (v6.2 and lower = root)
+- password = The same as your root login password. Default is dietpi
+- address = Your IP address (e.g.: 192.168.0.100)
+- port = 21
+
+### Options
+
+#### Destination directory
+
+The destination directory can be changed by replacing **/Path/To/Directory** to your target directory:
+
+```bash
+systemctl stop proftpd
+sed -i '/DefaultRoot /c\DefaultRoot /Path/To/Directory' /etc/proftpd/proftpd.conf
+systemctl start proftpd
+```
+
+#### Jailing (lock users to their home folders)
+
+*Jailing* can be enabled via
+
+```bash
+systemctl stop proftpd
+sed -i "/DefaultRoot /c\DefaultRoot ~" /etc/proftpd/proftpd.conf
+systemctl restart proftpd
+```
+
+See also <https://en.wikipedia.org/wiki/ProFTPD>.
+
+## Samba - Feature rich file server
+
+The Samba server lets you share files on your DietPi system with ease based on the well known SMB networking protocol.
+
+![DietPi file server software samba](../assets/images/dietpi-software-fileservers-samba.png)
+
+### Access to Samba
+
+The access to the Samba file server is achieved as follows:
+
+- Address = \\\192.168.0.100\dietpi or \\\dietpi\dietpi
```

Comment:
I think it needs to be `\\\DietPi\dietpi` since the default hostname is `DietPi`. But same as above, this can change so probably an abstraction like `yourHostname` or `<hostname>` makes sense.

Reply:
Changed.